core/src/main/java/org/opensearch/sql/utils/DateTimeFormatters.java

```diff
@@ -113,23 +113,25 @@ public class DateTimeFormatters {
 
   public static final DateTimeFormatter DATE_TIME_FORMATTER_VARIABLE_NANOS =
       new DateTimeFormatterBuilder()
-          .appendPattern("yyyy-MM-dd HH:mm:ss")
+          .appendPattern("uuuu-MM-dd HH:mm:ss")
```

Comment:
any impact on code or on user side that we switch from yyyy to uuuu?

Reply:
No, `yyyy` is the `year-of`era` and can be used in combination with a `G` to allow `AD` or `BC`. `uuuu` is just the year, user side should notice no difference. Change was made because the strict formatter (to cause exception when dates like February 30th are passed) requires `uuuu` instead of `yyyy`. 
https://stackoverflow.com/a/41178418

Reply:
I read that earlier but couldn't figure out. Thanks for explaining.

Reply:
`yyyy` rejects year less than Epoch (1970).